repos/spack_repo/builtin/packages/libmesh/package.py

```diff
@@ -126,6 +127,7 @@ class Libmesh(AutotoolsPackage):
         multi=False,
     )
     variant("shared", default=True, description="Enables the build of shared libraries")
+    variant("poly2tri", default=False, description="Compile the poly2tri library")
```

Comment:
@capitalaslash the description of this variant could be improved by changing it to describe that it changes from the viewpoint of a libmesh user/caller (which feature it enables). Simply a "Compile" could be done without the variant. A a minimum, it may say "compile libmesh with libpoli2tri" but, prefreably, it could better be "Enable poly2tri for ....".